### ISSUE TITLE
StyleCop applies rules if they are not present in custom ruleset file

### ISSUE THREAD

**Issue by prbans** [DotNetAnalyzers/StyleCopAnalyzers#2935]
I have the following ruleset file for testing purposes:

```
<?xml version="1.0" encoding="utf-8"?>
<RuleSet Name="prbans ruleset" Description="prbans ruleset for testing" ToolsVersion="15.0">
</RuleSet>
```

When I build, I notice the following errors (and more) in my codebase:

![image](https://user-images.githubusercontent.com/42591144/55926626-e713d680-5bc6-11e9-9403-afc2531b42a1.png)

I'd expect to see no warnings because there are no rules defined in my file to test against.

Couple of questions:
1) Is my assumption regarding no warnings expected as no rules defined correct? 
2) If not, is there any documentation about default applied rulesets if no custom behavior is defined?

I'm using StyleCop v1.1.1-rc.114

**Comment by vweijsters** [comment]
You're assumption is incorrect. AFAIK when a ruleset does not contain a rule, the default behaviour for that rule will be active. For StyleCop.Analyzers that means producing a warning.

Unfortunately, I can't find any documentation that describes this behavior.

**Comment by sharwell** [comment]
The behavior described by @vweijsters is correct. There is no way for us to change the behavior without breaking all existing users, so we are forced to close this as **won't fix**.
